actionmailer/lib/action_mailer/mail_helper.rb

```diff
@@ -4,7 +4,27 @@ module ActionMailer
   # attachments list.
   module MailHelper
     # Take the text and format it, indented two spaces for each line, and
-    # wrapped at 72 columns.
+    # wrapped at 72 columns:
+    #
+    #   text = <<-TEXT
+    #     This is the
+    #     first     paragraph.
+    #
+    #     The second
+    #        paragraph.
+    #
+    #     * item1 * item2
+    #       * item3
+    #   TEXT
+    #
+    #   block_format text
+    #   # =>  This is the first paragraph.
```

Comment:
Nit-picky thing but I guess you should either write:

~~~
block_format text
# => "  This is the first paragraph.\n\n  The second paragraph.\n\n  * item1\n  * item2\n  * item3\n"
~~~
or use a `puts` call and display the output removing all the extra `=>` but displaying the returned string seems more consistent with other methods' documentation (e.g. [`format_paragraph`](https://github.com/rails/rails/blob/ec8d8040a9c2d63c34ce0d3d3186c6fbb4230f37/actionmailer/lib/action_mailer/mail_helper.rb#L41-L42) or [`simple_format`](https://github.com/rails/rails/blob/ec8d8040a9c2d63c34ce0d3d3186c6fbb4230f37/actionview/lib/action_view/helpers/text_helper.rb#L261-L282))